streamer/src/nonblocking/quic.rs

```diff
@@ -130,17 +130,28 @@ pub fn spawn_server(
     max_streams_per_ms: u64,
     wait_for_chunk_timeout: Duration,
     coalesce: Duration,
-) -> Result<(Endpoint, Arc<StreamStats>, JoinHandle<()>), QuicServerError> {
+) -> Result<
+    (
+        Endpoint,
+        Arc<StreamStats>,
+        JoinHandle<()>,
+        u32, /* max_concurrent_connections */
```

Comment:
looks like we need a `struct` here.

Reply:
ya, just wanted to reduce the changes. but my comment indicates a need for a struct.